Assets/UI/worldviewiconsmanager.lua

```diff
@@ -338,7 +341,8 @@ function GetNonEmptyAt(plotIndex, state)
       -- Starting plot?
       if pPlot:IsStartingPlot() and WorldBuilder.IsActive() then
         pInstance = GetInstanceAt(plotIndex);
-        pInstance.RecommendationIconTexture:SetTexture( IconManager:FindIconAtlas("ICON_UNITOPERATION_FOUND_CITY", 43) );
+        -- CQUI updated 38->43 but 1.0.0.290 set it to 256
```

Comment:
I'd put this comment in the PR, not in the code. But that's my opinion, don't know what @Azurency thinks.

Reply:
Thanks - in affect you've now done this, will let him say which form he wants in future...